packages/office-ui-fabric-react/src/components/ChoiceGroup/examples/ChoiceGroup.Custom.Example.tsx

```diff
@@ -2,9 +2,23 @@ import * as React from 'react';
 import { ChoiceGroup } from 'office-ui-fabric-react/lib/ChoiceGroup';
 import { css } from 'office-ui-fabric-react/lib/Utilities';
 import { Dropdown } from 'office-ui-fabric-react/lib/Dropdown';
+import { mergeStyleSets } from '@uifabric/styling';
 
-import * as stylesImport from './ChoiceGroup.Custom.Example.scss';
-const styles: any = stylesImport;
+interface IChoiceGroupClassObject {
+  root: string;
+  dropdown: string;
+}
+
+const classNames: IChoiceGroupClassObject = mergeStyleSets({
```

Comment:
In this case it's probably best to use `mergeStyles` for the root and the `styles` prop for the dropdown.